packages/compass-welcome/src/components/web-welcome-tab.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import {
+  Button,
+  ButtonVariant,
+  H3,
+  spacing,
+  css,
+  Body,
+  Link,
+} from '@mongodb-js/compass-components';
+import { useConnectionIds } from '@mongodb-js/compass-connections/provider';
+import { WelcomeTabImage } from './welcome-image';
+
+const welcomeTabStyles = css({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  margin: '0 auto',
+  gap: spacing[200],
+});
+
+const contentBodyStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  gap: spacing[400],
+  alignItems: 'flex-start',
+});
+
+function getCreateAtlasClusterUrl() {
+  return window.location.href.replace(/#.*$/, '#/clusters/starterTemplates');
```

Comment:
You can just do `#/clusters/starterTemplates` here, you don't need a full url to replace a fragment

Reply:
added in [3b19e29](https://github.com/mongodb-js/compass/pull/6593/commits/3b19e296150181dcdfe88fc3efcf661e151f8fc7)

Reply:
i also removed more code in [f44ae52](https://github.com/mongodb-js/compass/pull/6593/commits/f44ae527000f21f6aa2e51d9d89fc530d8aa2629) to fix the test failures